utils/errors/errors.go

```diff
@@ -129,7 +139,7 @@ func packageFuncName(pc uintptr) (string, string) {
 }
 
 //New creates a new error with stack information
-func New(msg string) ErrorExt {
+func New(msg string, rawData ...interface{}) ErrorExt {
```

Comment:
I'm against changing the function signature for `New()`. Traditionally `.New()` accepts only a single string.

Reply:
Also rawData is unused in this function?